### PR TITLE
[WIP] Feature/mico14 restapi

### PR DESCRIPTION
**Pull request by Pascal157600**
- [x] Ensure that the commit message is [a good commit message](https://github.com/joelparkerhenderson/git_commit_message)
- [x] Ensure to use auto format in **all** files
- [ ] Tests created for changes or:
  - [ ] Test cases are missing - opened issue for that:
  - [ ] Reason for missing tests:
- [ ] Screenshots added (for UI changes)
- [ ] Change in CHANGELOG.md described

---

## Short Description

Add REST http methods for services

## Resolving Issue/ Feature

Fix Issue #14

### CODE REVIEW

**Comment by JanVetter** [comment]
**TODO**: We need to write tests.

**Comment by wagnerdk** [comment]
I think "getServiceByName" is wrong because it is not a noun. Our URLs should identify resources. Please consider using /services/ and then an id as a path parameter. If you want to enable searching you could use a query parameter.

**Comment by JanVetter** [comment]
Tests might be implemented like shown here: https://www.tutorialspoint.com/spring_boot/spring_boot_rest_controller_unit_test.htm

File: mico-core/src/main/java/io/github/ust/mico/core/ServiceController.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+    @Autowired
+    private ServiceRepository serviceRepository;
+
+    @GetMapping("/service")
+    public List<Service> getAllServices(){
+        return (List<Service>) serviceRepository.findAll();
+    }
+
+    @GetMapping(value = "/getServiceByName/{serviceName}")
```
[the remark this answers is not in the document]
Comment (Pascal157600) [reply mico-core/src/main/java/io/github/ust/mico/core/ServiceController.java]: I removed it.

**Pull request closed without merging**